docs/core/getting_started/basic-core-flow.md

```diff
@@ -0,0 +1,33 @@
+# Run a flow
```

Comment:
what is a flow?

perhaps we can provide a one sentence definition and link to a page for more detail?